taichi/ir/type.cpp

```diff
@@ -84,6 +84,26 @@ std::string PrimitiveType::to_string() const {
   return data_type_name(DataType(const_cast<PrimitiveType *>(this)));
 }
 
+size_t PrimitiveType::type_size() {
```

Comment:
Is https://github.com/taichi-dev/taichi/blob/4e6da3da971dbd630d2fb8878aea0f09530b8c4b/taichi/ir/type_utils.cpp#L30-L69 enough for this purpose?